### PR TITLE
Added to Thanks for contributing

### PR DESCRIPTION
**Pull request by saarthakchats**
Added French and Korean.

### CODE REVIEW

**Review by dilpreetsio — APPROVED** [review]

**Comment by saarthakchats** [comment]
Can you merge now.

**Review by vaibhavsingh97 — APPROVED** [review]

**Comment by anshumanv** [comment]
@saarthakchats you need to be patient, let other reviewers have their reviews.

**Review by anshumanv — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -204,7 +204,7 @@ <h4 class="uppercase mb16 animated fadeIn">Student Projects</h4>
 	 <div class="container">
 		<div class="col-md-10 col-md-offset-1 col-sm-7 col-sm-offset-1 v-align-transform text-center">
 		
-		 <h2 class="txt-rotate" data-period="1000" data-rotate='[ "Thank You For Contributing.", "योगदान करने के लिए धन्यवाद.", "Danke für den Beitrag.", "貢献していただきありがとうございます.", "பங்களித்ததற்கு நன்றி" ]'>
+		 <h2 class="txt-rotate" data-period="1000" data-rotate='[ "Thank You For Contributing.", "योगदान करने के लिए धन्यवाद.", "Danke für den Beitrag.", "貢献していただきありがとうございます.", "பங்களித்ததற்கு நன்றி", "merci de contribuer", "기여해 주셔서 감사합니다." ]'>
```
Comment (vaibhavsingh97) [inline index.html]: Korean translation is wrong change it to `공헌 해 주셔서 감사합니다.`

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (anshumanv) [reply index.html]: Thanks for verifying closely @vaibhavsingh97 :smile:

**Comment by vaibhavsingh97** [comment]
@saarthakchats Please resolve conflicts and we are ready to merge

**Comment by saarthakchats** [comment]
Resolved

File: index.html [hunk 2]
```diff
@@ -204,7 +204,11 @@ <h4 class="uppercase mb16 animated fadeIn">Student Projects</h4>
 	 <div class="container">
 		<div class="col-md-10 col-md-offset-1 col-sm-7 col-sm-offset-1 v-align-transform text-center">
 		
+
+		 <h2 class="txt-rotate" data-period="1000" data-rotate='[ "Thank You For Contributing.", "योगदान करने के लिए धन्यवाद.", "Danke für den Beitrag.", "貢献していただきありがとうございます.", "பங்களித்ததற்கு நன்றி", "merci de contribuer", "공헌 해 주셔서 감사합니다." ]'>
```
Comment (abishekvashok) [inline index.html]: Add it to the existing h2's data-rotate attribute pls

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
@anshumanv @abishekvashok Resolved the issues

File: index.html [hunk 3]
```diff
@@ -204,7 +204,11 @@ <h4 class="uppercase mb16 animated fadeIn">Student Projects</h4>
 	 <div class="container">
 		<div class="col-md-10 col-md-offset-1 col-sm-7 col-sm-offset-1 v-align-transform text-center">
 		
-		 <h2 class="txt-rotate" data-period="1000" data-rotate='[ "Thank You For Contributing.", "योगदान करने के लिए धन्यवाद.", "Danke für den Beitrag.", "貢献していただきありがとうございます.", "பங்களித்ததற்கு நன்றி", "Tak for bidrag.", "Grazie per aver contribuito.", "Go raibh maith agat as ranníoc." ]'>
+
```
Comment (vaibhavsingh97) [inline index.html]: Please remove extra whiteline space

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

**Comment by saarthakchats** [comment]
@vaibhavsingh97 
Now is it ready to approve?

**Comment by saarthakchats** [comment]
I mean merge?

**Comment by saarthakchats** [comment]
As you merge. Please also approve my GCI task

**Review by abishekvashok — APPROVED** [review]
LGTM

**Review by vaibhavsingh97 — APPROVED** [review]

**Pull request merged**